### DISCUSSION TITLE
I can't use viem in my nodejs backend

### DISCUSSION THREAD

**Discussion by fedecastelli** [wevm/viem#1238] [Question]
Hello,

I'm trying to use `viem` in my nodejs backend environment and I need to release the package to be available in my organization. I'm using `projen` to synthesize project configuration files such as package.json, tsconfig.json, .gitignore, GitHub Workflows, eslint, jest, etc from a well-typed definition written in JavaScript.

Once I run `npx projen build` (it's the same of running `yarn run build`), I'm getting these errors 

<img width="778" alt="image" src="https://github.com/wagmi-dev/viem/assets/31770652/922181e3-69a6-4e0a-b9b5-196bc368f1b5">

My `tsconfig.json` file has this configuration:

```json
{
  "compilerOptions": {
    ...
    lib: [
      "es2019"
    ]
  }
}
```

I read I should add `["es2017", "dom"]` to fix the error above. I don't think it's the right thing to do since "dom" is used to interact with HTML and XML on browsers. 

What do I have to do to fix the error? Is it safe to add "dom" in my `tsconfig.json` file?

**Comment by tmm** [comment]
Please share a link to a minimal reproduction. Without it, unlikely anyone is able to help.
